Calendar/CalLogHelpers/CalLogCSVFunctions.ps1

```diff
@@ -223,3 +219,35 @@ function ConvertDateTime {
     }
     return [DateTime]$DateTime
 }
+
+function GetAttendeeCount {
+    param(
+        [string] $AttendeeCollection
+    )
+    if ($From.SmtpAddress -ne "NotFound") {
+        return ($AttendeeCollection -split ';').Count
+    } else {
+        return "-"
+    }
+}
+
+function CompressedVersion {
+    [string] $CompressedString = ""
+    if ($CalLog.AppointmentSequenceNumber -eq "NotFound" -or [string]::IsNullOrEmpty($CalLog.AppointmentSequenceNumber)) {
```

Comment:
`$CalLog` is a local variable the function `BuildCSV` and `CompressedVersion` is not within that function. Therefore, we should pass the variable or move the function inside the `BuildCSV`